discovery.md

```diff
@@ -0,0 +1,378 @@
+# CloudSubscriptions: Discovery - Version 0.1
+
+## Abstract
+
+CloudSubscriptions Discovery API is a vendor-neutral API specification for
+determining what events an Event Producer or middleware (Aggregator, Broker)
+has available, as well as how to subscribe to those events.
+
+## Status of this document
+
+This is a working draft. Breaking changes could be made in the next minor
+version.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Notations and Terminology](#notations-and-terminology)
+- [API Specification](#api-specification)
+- [Protocol Bindings](#protocol-bindings)
+- [Privacy & Security](#privacy-and-security)
+
+## Overview
+
+The goal of the CloudDiscovery API is to enable connection between consumers of
+events and producers / aggregators / brokers of events and to facilitate the
+creation of CloudSubscriptions. The discovery and subscription APIs are
```

Comment:
I'd prefer if we didn't talk about control-plane vs data-plane aspects since I think that'll be an implementation detail. Our job is to just define the externals surface (APIs).